src/algebra/algebra/operations.lean

```diff
@@ -287,6 +287,27 @@ lemma le_div_iff {I J K : submodule R A} : I ≤ J / K ↔ ∀ (x ∈ I) (z ∈
 lemma le_div_iff_mul_le {I J K : submodule R A} : I ≤ J / K ↔ I * K ≤ J :=
 by rw [le_div_iff, mul_le]
 
+lemma one_le_inv {I : submodule R A} (hI : I ≤ (1 : submodule R A)) :
+  (1 : submodule R A) ≤ (1 : submodule R A) / I :=
```

Comment:
```suggestion
lemma one_le_inv {I : submodule R A} (hI : I ≤ 1) :
  (1 : submodule R A) ≤ 1 / I :=
```

Reply:
Lean reads from left to right : by the time it knows you're talking about inequalities of submodules (by reading the LHS) it assumes that the RHS has the same type.

Reply:
Ah, thanks, good to keep in mind.